corehq/messaging/scheduling/templates/scheduling/partials/conditional_alert_save.html

```diff
@@ -9,19 +10,19 @@
           <h4 class="modal-title">Conditional Alert</h4>
         </div>
         <div class="modal-body">
-          <p>This update may alert <span id="case-count"></span> cases. Are you sure you want to proceed?</p>
+          <p>{% trans "This update may alert" %} <span id="case-count"></span> {% trans "cases" %}. {% trans "Are you sure you want to proceed" %}?</p>
         </div>
         <div class="modal-footer">
-          <button type="button" class="btn btn-default" data-dismiss="modal" id="conditional-alert-yes-btn">Yes</button>
-          <button type="button" class="btn btn-default" data-dismiss="modal" id="conditional-alert-no-btn">No</button>
+          <button type="button" class="btn btn-primary" data-dismiss="modal" id="conditional-alert-yes-btn">{% trans "Yes" %}</button>
```

Comment:
In general, a user should only see one primary button at a time. In this case, most of the time we're expecting them to click yes, so yes should be primary and no can still be `btn-default`.